src/biogeochem/DryDepVelocity.F90

```diff
@@ -376,12 +381,12 @@ subroutine depvel_compute( bounds, &
             ! 4 - Winter, snow on ground and subfreezing
             ! 5 - Transitional spring with partially green short annuals
 
-
-            !mlaidiff=jan-feb
-            minlai=minval(annlai(:,pi))
-            maxlai=maxval(annlai(:,pi))
-
-            index_season = -1
+            if(.not. use_fates)then
```

Comment:
Why this conditions is removed?

Reply:
Do you mean minlai and maxlai? (they are now below in the if statement to make the logic clearer)...